app/ledger/neonLedger.js

```diff
@@ -226,7 +244,7 @@ export const getPublicKey = async (acct: number = 0): Promise<string> => {
 export const getDeviceInfo = async () => {
   const ledger = await NeonLedger.init()
   try {
-    return await ledger.getDeviceInfo()
+    return ledger.getDeviceInfo()
```

Comment:
no need for `await` here?

Reply:
I cleaned up the other instance of return await (that was in the code base before this PR) as `return await` is basically useless most of the time... If you are already returning a promise there is no point in awaiting its result (unless there is a catch block to catch errors from another Promise). More info can be found here even though we dont have this rule configured currently https://eslint.org/docs/rules/no-return-await

Reply:
Looks like by removing the `return await` from `getPublicKeys` the functionality breaks totally puzzled by that one 😕

Reply:
Perhaps without the `await`, the `finally` executes `ledger.close()` immediately, therefore not allowing `getDeviceInfo()` to happen 🤔